libs/SoloPlay/Functions/MiscOverrides.js

```diff
@@ -464,7 +464,7 @@ Misc.getSocketables = function (item, itemInfo) {
 
 Misc.checkSocketables = function () {
 	let items = me.getItemsEx()
-		.filter(item => item.getStat(sdk.stats.NumSockets) > 0 && AutoEquip.hasTier(item) && ![sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality)) // Normal Quality & Superior Quality
+		.filter(item => item.getStat(sdk.stats.NumSockets) > 0 && AutoEquip.hasTier(item) && ![item.normal, item.superior].includes(item.quality))
```

Comment:
```[sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality))```
remember what I said about numerical vs boolean.
item.quality === sdk.itemquality.Magic // is boolean its just checking if the values equal each other
[sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality)) is equal to the statement
[2, 3].includes(item.quality) // where item.quality is returning the actual value and we are seeing whether the array includes that value

Reply:
my bad, I did a bad copy/paste